heic2jpg.sh

```diff
@@ -2,15 +2,22 @@
 
 src=$1
 size=$(find $src -type f | grep -i '\.heic$' | wc -l)
-cnt=0
+iter=0
 
-for i in $(find $src -type f | grep -i '\.heic$')
+for fn in $(find $src -type f | grep -i '\.heic$')
 do
-	((cnt=cnt+1))
-	target=$(echo $i | sed 's/heic$/jpg/i')
-	printf "$i convert to $target\n"
-	/usr/bin/convert $i $target
-	printf " [$cnt/$size]\r"
+	iter=$((iter+1))
+	target=$(echo $fn | sed 's/heic$/jpg/i')
+	# https://www.tutorialkart.com/bash-shell-scripting/bash-date-format-options-examples/
+	FILE_DATE=$(/bin/date +%Y%m%d%H%M.%S -r ${fn})
```

Comment:
Couldn't we get the date&time from the EXIF instead? That would make more sense to me than relying on the file system itself.

Reply:
In different script of mine, I successfully used the `exiv2` tool for similar thing.

In this case running `exiv2 -T -F rename *.jpg` would do the trick. Wouldn't it be worth adding this to the docker image?

Reply:
The file date is used by other photos management apps. Let's make sure we set the file date as the original created date, or use EXIF.

Reply:
I don't know if you got me right. I think the creation date of JPG file should be set from EXIF, rather than copying it from creation date of HEIF file.

That is exactly what `exiv2 -T -F rename *.jpg` is doing. It will go thru all *.jpg files, reads the date from EXIF, and sets it to the file.

Reply:
I agree.

what I'm saying is that it depends of the photo management system. There are two places where the date is stored. Some uses the filesystem creation date, others from EXIF.  Let's make both of thrm have the same date. 

Let's make the converted file to have the same date at both places. AVOID having the convertdd file to have created  date of today.

I'll adjust my PR